clients/admin-ui/src/features/datamap/reporting/DatamapReportTable.tsx

```diff
@@ -0,0 +1,985 @@
+        header: (props) => (
+          <DefaultHeaderCell value="Uses cookies" {...props} />
+        ),
+        meta: {
+          width: cellWidth,
+          displayText: "Uses cookies",
+        },
+      }),
+      columnHelper.accessor((row) => row.uses_non_cookie_access, {
+        id: COLUMN_IDS.USES_NON_COOKIE_ACCESS,
+        cell: (props) => <DefaultCell value={props.getValue()} />,
+        header: (props) => (
+          <DefaultHeaderCell value="Uses non cookie access" {...props} />
+        ),
+        meta: {
+          width: cellWidth,
+          displayText: "Uses non cookie access",
+        },
+      }),
+      columnHelper.accessor((row) => row.uses_profiling, {
+        id: COLUMN_IDS.USES_PROFILING,
+        cell: (props) => <DefaultCell value={props.getValue()} />,
+        header: (props) => (
+          <DefaultHeaderCell value="Uses profiling" {...props} />
+        ),
+        meta: {
+          width: cellWidth,
+          displayText: "Uses profiling",
+        },
+      }),
+    ],
+    []
+  );
+
+  const {
+    isOpen: isColumnSettingsOpen,
+    onOpen: onColumnSettingsOpen,
+    onClose: onColumnSettingsClose,
+  } = useDisclosure();
+
+  const tableInstance = useReactTable<DatamapReport>({
+    getCoreRowModel: getCoreRowModel(),
+    getGroupedRowModel: getGroupedRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    columns: tcfColumns,
+    manualPagination: true,
+    data,
+    initialState: {
+      columnOrder,
+    },
+    state: {
+      expanded: true,
+      grouping,
+    },
+  });
+
+  const getMenuDisplayValue = () => {
+    switch (groupBy) {
+      case DATAMAP_GROUPING.SYSTEM_DATA_USE: {
+        return "system";
+      }
+      case DATAMAP_GROUPING.DATA_USE_SYSTEM: {
+        return "data use";
+      }
+      case DATAMAP_GROUPING.DATA_CATEGORY_SYSTEM: {
+        return "data category";
+      }
+      default: {
+        return "system";
+      }
+    }
+  };
+
+  if (isReportLoading || isLoadingHealthCheck) {
+    return <TableSkeletonLoader rowHeight={36} numRows={15} />;
+  }
+
+  return (
+    <Flex flex={1} direction="column" overflow="auto">
+      <DatamapReportFilterModal
+        isOpen={isOpen}
+        onClose={onClose}
+        resetFilters={resetFilters}
+        dataUseOptions={dataUseOptions}
+        onDataUseChange={onDataUseChange}
+        dataCategoriesOptions={dataCategoriesOptions}
+        onDataCategoriesChange={onDataCategoriesChange}
+        dataSubjectOptions={dataSubjectOptions}
+        onDataSubjectChange={onDataSubjectChange}
+      />
+      <ColumnSettingsModal<DatamapReport>
+        isOpen={isColumnSettingsOpen}
+        onClose={onColumnSettingsClose}
+        prefixColumns={getPrefixColumns(groupBy)}
+        tableInstance={tableInstance}
+      />
+      <TableActionBar>
+        <GlobalFilterV2
+          globalFilter={globalFilter}
+          setGlobalFilter={updateGlobalFilter}
+          placeholder="Search"
```

Comment:
I'll leave this one up to you since this behavior is up in the air still, but how about changing the placeholder to "System name, Fides key, or ID"?
![image](https://github.com/ethyca/fides/assets/2092272/df00dbd6-cf46-4801-9b6f-d21e85859139)

Reply:
I'll update it :+1: